README.md

```diff
@@ -134,6 +134,21 @@ Analyzer is installed. For example, on Windows the default installation
 location is C:\\Program
 Files\\Fortify\\Fortify\_SCA\_and\_Apps\_*\<version\>*.
 
+### Configuration when running Jenkins from the Docker
+
+When running Jenkins from under the Docker container you need to mount *\<sca\_install\_dir\>* 
```

Comment:
'under the Docker container you need to mount' -> 'a Docker container, mount'

Reply:
Done.